test/integration/store_and_character_have_items_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.